### DISCUSSION TITLE
Does Airbyte support Keycloak Authentication & Authorization?

### DISCUSSION THREAD

**Discussion by lethehoa** [airbytehq/airbyte#32339] [Connector Questions] [answered]
Hello everyone, Is Airbyte UI support keycloak authentication & authorization or any other ways to achieve those need. I installed Airbyte in K8S cluster with helm. I've read the document but this kind of hard to understand.
Thanks for you help.

**Comment by marcosmarxm** [comment, marked as answer]
Yes, but it is only supported for [Airbyte Enterprise](https://airbyte.com/solutions/airbyte-enterprise).
You can find more information about the future of auth for community version in this [issue](https://github.com/airbytehq/airbyte/issues/30360)
